pair-filter/pair-filter.py

```diff
@@ -110,6 +115,10 @@ def _unfiltered_key(pr: bool):
                 elif reason == reasons.INACCESSIBLE_IMAGE:
                     d[_key('inaccessible_image', is_pr)] += 1
         for metric_name, metric_value in d.items():
+            try:
+                metric_value = metric_value + original_d['progression_metrics'][metric_name]
+            except KeyError:
+                pass
```

Comment:
When would we get a `KeyError` and why do we want to pass?

Reply:
We would get a `KeyError` when the project has not yet been mined (meaning only the 1st time mining), and we want to pass through so the metrics could be set. The next time you mine the same project, dictionary variable `original_d` will query for the metrics from the DB to be used for adding metrics.